projects/plugins/jetpack/extensions/shared/memberships/settings.scss

```diff
@@ -75,3 +75,12 @@
 	}
 
 }
+
+.editor-post-tiers {
```

Comment:
A bit generic top level selector — would be good to prefix with `jetpack-`

Reply:
Done